pkg/clusterdiscovery/clusterapi/clusterapi.go

```diff
@@ -27,6 +28,14 @@ import (
 	"github.com/karmada-io/karmada/pkg/util/informermanager/keys"
 )
 
+func init() {
+	flag.IntVar(&concurrentReconciles, "clusterapi-workers", concurrentReconciles, "Max concurrent workers for clusterapi worker.")
+}
+
+var (
+	concurrentReconciles = 5
+)
```

Comment:
Personally, I don't like using `init` to add flags, the reasons are:
- We can't easily share code as the flag will be added automatically.
- It'll lose the chance to validate the input.

Reply:
ok,let's move this into options.